### ISSUE TITLE
New Zealand Parliament

### ISSUE THREAD

**Issue by Frostieyyy** [firlus/parliament-simulator#22]
Would it be possible to add the New Zealand Parliament into this?

**Comment by firlus** [comment]
@Frostieyyy Yeah sure, I didn't update the data for this for a long time but will probably do so in the next few days. I will also include New Zealand :)
